### PR TITLE
docs(viewport): fix spec URL for viewport `<meta>`

### LINKED ISSUE #20982: Broken spec link
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag

### What specific section or headline is this issue about?

Specification 

### What information was incorrect, unhelpful, or incomplete?

The link provided for the[ specification](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#specifications) related to the information provided on the page is broken.

### What did you expect to see?

A proper working link for the specification reference.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/viewport_meta_tag`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/viewport_meta_tag/index.md
* Last commit: https://github.com/mdn/content/commit/8e1f45b8c0a3671f6e3de43c0764ad610aa345dd
* Document last modified: 2022-09-14T09:06:12.000Z

</details>

### PR DESCRIPTION
**Pull request by bsmth**
It looks like the following changes should be made:

* Old URL https://drafts.csswg.org/css-device-adapt/#viewport-meta (broken)
* New URL https://www.w3.org/TR/css-device-adapt-1/#viewport-meta

Added spec URL to `SpecData.json` and in related doc.

Fixes #20982

__Open question:__

Do we remove the old URL from `SpecData.json` or any references to it?

### CODE REVIEW

**Comment by dipikabh** [comment]
The [guidelines](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Document_a_CSS_property#step_1_%E2%80%94_determine_the_property_to_document) recommend to use the Editor's draft.
What do you think about replacing the link with: https://drafts.csswg.org/css-viewport/#viewport-meta?

File: files/jsondata/SpecData.json [hunk 1]
```diff
@@ -254,6 +254,11 @@
     "url": "https://drafts.csswg.org/css-device-adapt/",
     "status": "WD"
   },
+  "CSS Device Adaptation Module Level 1": {
+    "name": "CSS Device Adaptation Module Level&nbsp;1",
+    "url": "https://www.w3.org/TR/css-device-adapt-1/",
+    "status": "ED"
+  },
```
Comment (dipikabh) [inline files/jsondata/SpecData.json:261]: In ED, this is now called "CSS Viewport Module Level 1".
I have the same question as you, so not much help - does "Device Adaptation Module" from SpecData.json need to be removed?

Comment (bsmth) [reply files/jsondata/SpecData.json:261]: The link to the old spec is broken, so I would remove it and update the new URL like you mention.

**Comment by bsmth** [comment]
> The [guidelines](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Document_a_CSS_property#step_1_%E2%80%94_determine_the_property_to_document) recommend to use the Editor's draft.
> What do you think about replacing the link with: https://drafts.csswg.org/css-viewport/#viewport-meta?

Let me do that, thanks for pointing it out 👍🏻

File: files/jsondata/SpecData.json (hunk 1 above)
Comment (bsmth) [reply files/jsondata/SpecData.json:261]: >In ED, this is now called "CSS Viewport Module Level 1".

In https://www.w3.org/TR/css-device-adapt-1/ it looks like "CSS Device Adaptation Module Level 1", is that correct?

**Comment by schalkneethling** [comment]
> It looks like we have a duplicate at #21018

Dang, I merged the other one before seeing yours. Thanks, @bsmth.

**Comment by schalkneethling** [comment]
@bsmth, I see you have a couple of additional fixes, this one just needs a rebase to address conflicts.

File: files/jsondata/SpecData.json (hunk 1 above)
Comment (dipikabh) [reply files/jsondata/SpecData.json:261]: It'll need to be replaced with "CSS Viewport Module Level 1".

**Comment by bsmth** [comment]
I'm going to close this one for now because the Editor's Draft is not available. We will revisit when https://github.com/mdn/yari/pull/7175 is merged

**Pull request closed without merging**